test_collections/matter/scripts/update-sample-apps.sh

```diff
@@ -43,7 +43,7 @@ fi
 
 
 print_script_step "Updating Sample APPs"
-sudo docker run -t -v ~/apps:/apps $SDK_DOCKER_IMAGE bash -c "rm -v /apps/*; cp -v apps/* /apps/"
+sudo docker run -t -v ~/apps:/apps -v ~/mock_server:/mock_server $SDK_DOCKER_IMAGE bash -c "rm -v /apps/*; cp -v apps/* /apps/; cp -v -r mock_server/* /mock_server/"
```

Comment:
Should we remove the existing `mock_server` folder as we do for the apps folder as shown below?

```suggestion
sudo docker run -t -v ~/apps:/apps -v ~/mock_server:/mock_server $SDK_DOCKER_IMAGE bash -c "rm -v /apps/*; rm -v /mock_server/*; cp -v apps/* /apps/; cp -v -r mock_server/* /mock_server/"
```

Reply:
Nice catch. Thanks